.github/workflows/source/makeMakefileForClangTidy.py

```diff
@@ -14,40 +14,49 @@
 
 def makeMakefileForClangTidy(argv):
     parser = argparse.ArgumentParser()
-    parser.add_argument("--input",
-                        help="Ccache log file",
-                        default="ccache.log.txt")
-    parser.add_argument("--identifier",
-                        help="Unique identifier for finding compilation line in the log file",
-                        default="WarpX/Source")
+    parser.add_argument("--input", help="Ccache log file", default="ccache.log.txt")
+    parser.add_argument(
```

Comment:
If I were formatting this, I would do this, which I think is easier to read (maybe I'm just more used to it)
```
parser.add_argument("--identifier",
                    help = "Unique identifier for finding compilation line in the log file",
                    default = "WarpX/Source")
```
The arguments aligned with the routine name, and a space before and after the `=`.

Reply:
Thanks, this is auto-formatted using black style, as we do in all other projects already:
https://docs.astral.sh/ruff/formatter/